bin/plugin/commands/common.js

```diff
@@ -46,7 +46,7 @@ function calculateVersionBumpFromChangelog(
 	let changesDetected = false;
 	let versionBump = null;
 	for ( const line of lines ) {
-		const lineNormalized = line.toLowerCase().trimLeft();
+		const lineNormalized = line.toLowerCase().trimStart();
```

Comment:
This is replacing a deprecated `String` method (alias) with its recommended replacement.